src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/actions/PersistedAction.java

```diff
@@ -0,0 +1,94 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.persistence.actions;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.persistence.ValueColumn;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Action summary that can be persisted in SQL
+ *
+ * <p>Table name : PersistedAction
+ *
+ * <p>schema :
+ * | id(primary key) |     actionName       |   timestamp       |       nodeIds          |
+ * |      1          | ModifyQueueCapacity  |  1599257910923    |   node1, node2         |
+ * |        nodeIps         | actionable | coolOffPeriod    |  muted    |   summary         |
+ * | 127.0.0.1, 127.0.0.2   |  1         |  300             |   0       | actionSummary     |
+ */
+public class PersistedAction {
```

Comment:
nit: should we make this a nested static class in `PublisherEventsPersistor` ?

Reply:
During read(), the persistable (object used to persist things in sqlite) needs this class as it return as Class type. 
It is better to have this when write and read occur simultaneously.